core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -158,7 +158,7 @@ object NodepoolStatus {
 final case class KubernetesClusterLeoId(id: Long) extends AnyVal
 final case class NamespaceId(id: Long) extends AnyVal
 final case class Namespace(id: NamespaceId, name: NamespaceName)
-final case class KubernetesServiceAccount(value: String) extends AnyVal
+final case class KubernetesServiceAccountName(value: String) extends AnyVal
```

Comment:
changing this because we already have object `KubernetesServiceAccount` (which has both the SA account and its annotations) in wb-libs `kubernetesModels`

Reply:
Wonder if we should just use [ServiceAccountName](https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/kubernetesModels.scala#L129) in Leo..

Reply:
I think that's better, less to add to `kubernetesModels`